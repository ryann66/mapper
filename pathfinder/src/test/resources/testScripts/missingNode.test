# Attempt to find a path with missing nodes

CreateGraph mg

FindPath mg A B

AddNode mg A

FindPath mg A B

AddNode mg B

FindPath mg B C