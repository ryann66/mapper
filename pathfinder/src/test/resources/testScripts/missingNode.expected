# Attempt to find a path with missing nodes

created multigraph mg

unknown: A
unknown: B

added node A to mg

unknown: B

added node B to mg

unknown: C